.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "env": {
+    "commonjs": true,
+    "es2020": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 2018
```

Comment:
https://eslint.org/docs/user-guide/getting-started#installation-and-usage

running `npx eslint --init` initialized the `ecmaVersion` to `12` but my vscode was showing some kind of warning, so i downgraded. dunno what to do about that fyi.